tracer/test/Datadog.Trace.Security.IntegrationTests/Rcm/AspNetCore5AsmToggle.cs

```diff
@@ -114,12 +114,6 @@ private void CheckAckState(GetRcmRequest request, uint expectedState, string exp
             state.ApplyState.Should().Be(expectedState, message);
             state.ApplyError.Should().Be(expectedError, message);
         }
-
-        private void CheckCapabilities(GetRcmRequest request, uint expectedState, string message)
```

Comment:
Does the test still compile? It looks like this method is still being used

Reply:
Whoops, I noticed that you moved this to the base class. Nevermind, looks good 😄